0_From_Code_First_To_First_Principles/1_code_first/assignment/src/UserController.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from 'express';
+import { userView } from './UserView';
+import { UserModel } from './UserModel';
+
+const userModel = new UserModel();
+
+export class UserController {
+  async createUser(req: Request, res: Response) {
+    const { error, data, success, status } = await userModel.createUser(req.body);
+    res.status(status).json(userView(error, data, success));
```

Comment:
As to your question - it does make more sense to call the userView here. The UserModel (service) is not supposed to know about it at all. It's supposed to handle business logic only.